### PR TITLE
Correcting and adding info to NLP-OSS workshop 2023 (`2023.nlposs.xml`)

### PR DESCRIPTION
**Pull request by alvations**
- Removed additional authors from https://aclanthology.org/2023.nlposs-1.26/ 
- Added abstracts to the invited talks.

### CODE REVIEW

**Comment by alvations** [comment]
@acl-org-github , there's a mistake in the pdfs for 

 - https://aclanthology.org/2023.nlposs-1.26/
 - https://aclanthology.org/2023.nlposs-1.27/

The pdfs were wrongly swapped. How do we make the changes to correct it to the right pdfs?

**Comment by alvations** [comment]
Also another related questions, we have slides and posters for almost all the papers https://github.com/nlposs/NLP-OSS/tree/master/nlposs-2023, is there some pointers/guide to add them to the anthology as links, like how it's done for https://aclanthology.org/2023.nlposs-1.10/ ?

**Comment by mbollmann** [comment]
Currently we only support locally hosted attachments. But since they're all collected in one GitHub repo, maybe it's not so difficult to bulk-add them @anthology-assist ?

File: data/xml/2023.nlposs.xml [hunk 1]
```diff
@@ -288,28 +288,27 @@
...
       <url hash="b8184f0c">2023.nlposs-1.27</url>
       <bibkey>castricato-2023-trlx</bibkey>
     </paper>
     <paper id="28">
       <title>Towards Explainable and Accessible <fixed-case>AI</fixed-case></title>
-      <author><first>Brandon</first><last>Duderstadt</last><affiliation>Online University (can leave it as it will not shown)</affiliation></author>
-      <author><first>Yuvanesh</first><last>Anand</last><affiliation>Online University (can leave it as it will not shown)</affiliation></author>
+      <author><first>Brandon</first><last>Duderstadt</last><affiliation>Nomic AI</affiliation></author>
+      <author><first>Yuvanesh</first><last>Anand</last><affiliation>Virginia Institute of Technology & Nomic AI</affiliation></author>
```
Comment (mjpost) [inline data/xml/2023.nlposs.xml:309]: XML requires this to be written as `&amp;`

**Comment by mjpost** [comment]
The aclpub2 format has an option for attachments. If you add them to the repo and link them from `output/inputs/papers.yml` I can reingest.

**Comment by mjpost** [comment]
I swapped the PDFs

File: data/xml/2023.nlposs.xml (hunk 1 above)
Comment (alvations) [reply data/xml/2023.nlposs.xml]: Thanks! Fixed that in the new commit.

**Comment by alvations** [comment]
> The aclpub2 format has an option for attachments. If you add them to the repo and link them from `output/inputs/papers.yml` I can reingest.

Do you mean to include the attachments in the `papers.yml` in the compiled .tar.gz from the https://github.com/rycolab/aclpub2 instructrions? 

Or is it `papers.yml` somewhere directly in the https://github.com/acl-org/acl-anthology/ repo?

**Comment by mjpost** [comment]
There should be an attachments directory [here](https://github.com/emnlp-2023/W7-NLP-OSS/tree/main/output), and [papers.yml](https://github.com/emnlp-2023/W7-NLP-OSS/blob/main/output/inputs/papers.yml) should link to them.

**Comment by alvations** [comment]
I'm trying to understand the syntax of the `papers.yml` but I can't seem to find how/where the poster pdf link is added, e.g. for the ACL Anthology paper, I see this:

```
- abstract: The ACL Anthology is a prime resource for research papers within computational
    linguistics and natural language processing, while continuing to be an open-source
    and community-driven project. Since Gildea et al. (2018) reported on its state
    and planned directions, the Anthology has seen major technical changes. We discuss
    what led to these changes and how they impact long-term maintainability and community
    engagement, describe which open-source data and software tools the Anthology currently
    provides, and provide a survey of literature that has used the Anthology as a
    main data source.
  attributes:
    paper_type: N/A
    presentation_type: N/A
    submitted_area: null
  authors:
  - dblp_id:  ...
  decision: Accept
  end_page: 94
  file: 17.pdf
  id: 17
  num_pages: 12
  openreview_id: cZpZhNbbHV
  pdf_file: 28f69640e8fcc4d2f8a71522b4c5f1cb4384d561.pdf
  start_page: 83
  title: 'Two Decades of the ACL Anthology: Development, Impact, and Open Challenges'
```


Which contains: 

 - `17.pdf` is from https://github.com/emnlp-2023/W7-NLP-OSS/blob/main/output/inputs/papers/17.pdf 
 - `28f69640e8fcc4d2f8a71522b4c5f1cb4384d561.pdf` is from https://openreview.net/pdf/28f69640e8fcc4d2f8a71522b4c5f1cb4384d561.pdf

But I can't find where to add https://aclanthology.org/attachments/2023.nlposs-1.10.Poster.pdf to the yml file. Any pointer to where to add the poster pdf files in `papers.yml`?

**Comment by mjpost** [comment]
You should have communication from ACL pub chairs?

* Doc is here: https://github.com/rycolab/aclpub2#Expected-output
* checker script is here: https://github.com/acl-org/acl-anthology/blob/master/bin/anthology_check.py

**Comment by alvations** [comment]
Ah, it's the ` attachments:`. Thanks, I'll look at https://github.com/rycolab/aclpub2#Expected-output more carefully.

**Comment by mjpost** [comment]
Any updates?

**Review by mjpost — APPROVED** [review]

**Comment by mjpost** [comment]
Hi, I just went ahead and fixed some of these errors myself. If you want to add the attachments, we'd welcome them.

**Pull request merged**